examples/basic_routing/main.tf

```diff
@@ -1,29 +1,45 @@
-# ---------------------------------------------------------------------------------------------------------------------
+# ------------------------------------------------------------------------------
 # CREATE A ROUTE53 ZONE WITH SUBDOMAINS AND CNAMES
 # This example creates a zone and records for the main domain and a subdomain.
 #   - (www.)acme.com
 #   - (www.)dev.acme.com
 #
 # The www. subdomains are implement through CNAMES and point on the A records.
-# ---------------------------------------------------------------------------------------------------------------------
+# ------------------------------------------------------------------------------
 
-# ---------------------------------------------------------------------------------------------------------------------
+# ------------------------------------------------------------------------------
+# Set Terraform and Provider Requirements for running this example
+# ------------------------------------------------------------------------------
+
+terraform {
+  required_version = ">= 0.12.20"
+
+  required_providers {
+    aws = ">= 2.45"
+  }
+}
+
```

Comment:
we already discussed this and it was fixed by you... latest good one before the breaking force-push and merge seems to be https://github.com/mineiros-io/terraform-aws-route53/commit/aa7d9ca459a28a214c0fcb0d0b5c4a6ded499bbb